lib/utils.py

```diff
@@ -286,8 +286,8 @@ def xlog_rows(xlog_path):
     cmd = ['tarantoolctl', 'cat', xlog_path, '--format=json', '--show-system']
     with open(os.devnull, 'w') as devnull:
         process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=devnull)
-    for line in process.stdout.readlines():
-        yield json.loads(line)
+        for line in process.stdout.readlines():
+            yield json.loads(bytes_to_str(line))
```

Comment:
The stderr pipe (one that points to `/dev/null`) will be referenced by the Popen instance and the file descriptor will be closed by the garbage collector. Since we don't use it directly in those two lines, I think it is okay (and even logical) to keep them outside of the `with` block.

Reply:
Done